config/routes.rb

```diff
@@ -27,6 +27,8 @@
   namespace :v0, defaults: { format: 'json' } do
     get '/calculator/constants' => 'calculator_constants#index'
 
+    get '/institutions/:id/children/' => 'institutions#children'
```

Comment:
Can we move this into the `resources :institutions` block below?
`get :children, on: :member`